src/AppInstallerRepositoryCore/Microsoft/Schema/1_4/Interface_1_4.cpp

```diff
@@ -66,6 +67,18 @@ namespace AppInstaller::Repository::Microsoft::Schema::V1_4
         savepoint.Commit();
     }
 
+    bool Interface::NotNeeded(const SQLite::Connection& connection, std::string_view tableName, std::string_view valueName, SQLite::rowid_t id) const
```

Comment:
I can't put a comment on `RemoveManifestById`, but it isn't removing the versions that are no longer needed.